docs/css/microinteraction.md

```diff
@@ -0,0 +1,142 @@
+---
+id: microinteraction
+title: What is a microinteraction?
+sidebar_label: Microinteraction
+---
+
+import EditedBy from '../../src/components/EditedBy'
+
+According to Dan Saffer, author of the highly-respected book: Microinteractions: Designing with Details, a microinteraction is: “A contained product moment that revolves around a single use case or task.” He uses the examples of ‘liking a post’ or ‘setting an alarm’ as common microinteractions that most of us do every day. Saffer also explains in his book that microinteractions can be broken down into a 4 part structure:
```

Comment:
Use blockquotes in markdown. Put the quote part in a new line and prefix a ">" in front:
> A contained product moment that revolves around...

Reply:
Done